src/test/java/gov/usgs/wma/waterdata/groundwater/S3BucketUtilTest.java

```diff
@@ -77,34 +77,29 @@ void testFilenamePROD() {
 	}
 
 	@Test
-	void testOpenS3File() {
+	void testOpenS3File() throws Exception {
 		// SETUP
 		String filename = "test-filename";
 
 		// ACTION UNDER TEST
-		S3Bucket s3 = s3util.openS3(filename);
+		S3Bucket s3OuterRef = null;
 
-		try {
+		try (S3Bucket s3 = s3util.openS3(filename)) {
+
+			s3OuterRef = s3;    //Save a reference to check if it was closed
 			// ASSERTIONS
 			assertNotNull(s3);
 			assertEquals(properties.bucket, s3.bucket);
 			assertEquals(properties.region, s3.region);
 			assertEquals(filename+".gz", s3.keyName);
 			assertNotNull(s3.file);
+			assertTrue(s3.file.exists());
 
-			String tmpPath = s3.file.getAbsolutePath().toLowerCase();
-			assertTrue(tmpPath.contains("tmp") || tmpPath.contains("temp"));
-
-			// default state is to dispose of the tmp file
-			assertTrue(s3.isFileDisposable);
 			// initial state is without a writer
 			assertNull(s3.writer);
-		} finally {
-			// CLEANUP
-			if (s3.file != null) {
-				s3.file.delete();
-			}
```

Comment:
This finally block was actually handling the exact thing that AutoCloseable was intended to do, so I removed it and test to make sure AutoCloseable actually did what is was supposed to do instead (line 102).